website/www/site/data/io_matrix.yaml

```diff
@@ -397,3 +409,12 @@ categories:
           - language: java
             name: org.apache.beam.sdk.io.gcp.healthcare.HL7v2IO
             url: https://beam.apache.org/releases/javadoc/current/org/apache/beam/sdk/io/gcp/healthcare/HL7v2IO.html
+      - transform: Firestore IO
+        description: FirestoreIO provides an API for reading from and writing to Google Cloud Firestore.
+        implementations:
+          - language: java
+            name: org.apache.beam.sdk.io.gcp.healthcare.FirestoreIO
+            url: https://beam.apache.org/releases/javadoc/2.36.0/org/apache/beam/sdk/io/gcp/firestore/FirestoreIO.html
+      - transform: Neo4j
+        implementations:
+          - language: java
```

Comment:
Please move these Connectors to the `Database` table instead of `Miscellaneous`.